codespell_lib/data/dictionary.txt

```diff
@@ -1239,12 +1239,15 @@ amung->among
 amunition->ammunition
 analagous->analogous
 analgous->analogous
+analised->analysed
+analiser->analyser
 analises->analysis, analyses,
+analising->analysing
 analitic->analytic
 analitical->analytical
 analitically->analytically
 analiticaly->analytically
-analize->analyze
+analize->analyze, analyse,
```

Comment:
Why did you add `analyse` ? :man_facepalming:

Reply:
Woops, sorry. Forgot to remove that one